### PR TITLE
Fixes #89

### PR DESCRIPTION
**Pull request by ishanisri**

### CODE REVIEW

**Comment by PaiAkshay998** [comment]
Some description of what has been done / what this does? The issue is another repo so you can't reference a pull request from this repo to another repo.

**Comment by ishanisri** [comment]
Issue solved : It shows a blink in the order book book indicate that a transaction happened.
Note - It shows the blink when the new table is rendered

**Comment by PaiAkshay998** [comment]
Isn't working as expected. Check it out

File: public/src/components/trading_terminal/OrderBook/MarketDepth.tsx [hunk 1]
```diff
@@ -80,13 +79,13 @@ export class MarketDepth extends React.Component<MarketDepthProps, {}> {
             }
             buyRows.push(
                 <tr key={i}>
-                    <td className="volume"><strong>{typeof(bidArray[i]) != "undefined" ? bidArray[i].volume : ''}</strong></td>
-                    <td className="price green"><strong>{typeof(bidArray[i]) != "undefined" ? bidArray[i].price : ''}</strong></td>
-                    <td className="volume"><strong>{typeof(askArray[i]) != "undefined" ? askArray[i].volume : ''}</strong></td>
+                    <td className="volume buy" id={"buy-volume"+i}><strong>{typeof(bidArray[i]) != "undefined" ? bidArray[i].volume : ''}</strong></td>
```
Comment (PaiAkshay998) [inline public/src/components/trading_terminal/OrderBook/MarketDepth.tsx]: It's almost never a good idea to concat using ```+```. Works most of the time but there'll always be some weird case. Most languages will provide some sort of templating method to concat integers and strings which is more readable and at the same time works as expected.

**Comment by PaiAkshay998** [comment]
> Issue solved : It shows a blink in the order book book indicate that a transaction happened.
> Note - It shows the blink when the new table is rendered

Try to be a bit more descriptive in the sense, give a rough idea how you've gone about it. Something like

> Adding an id to each entry in the OrderBook and on every update received, adding css for the blinking part

Not really necessary all the time but it makes the reviewer's life a bit easy.

File: public/src/css/dalal.css [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+
+@-webkit-keyframes background {
+  0%,49% { background-color: grey; }
+  50%,100% { background-color:  #1B1C1D; }
+}
+
+@keyframes background {
+	0%,49% { background-color: grey; }
+	50%,100% { background-color:  #1B1C1D; }
+
```
Comment (PaiAkshay998) [inline public/src/css/dalal.css]: ??

File: public/src/css/dalal.css [hunk 3]
```diff
@@ -411,6 +427,7 @@ h3.panel-header {
 }
 #trading-terminal table tr td, #trading-terminal table tr td {
 	border-bottom: none !important;
+
```
Comment (PaiAkshay998) [inline public/src/css/dalal.css]: Why?

**Pull request merged**